app/utils/apiUtils.js

```diff
@@ -7,12 +7,15 @@ const apiClients = {
   github: null,
   default: null
 };
-export const getApiClient = (type = 'github') => apiClients[type];
+export const getApiClient = (type = 'github' || 'itunes') => apiClients[type];
```

Comment:
so type = 'github' just gives it a default value if type is not passed in. 

we wouldn't need to do `type = 'github' || 'itunes'`